src/components/GenreSelect.js

```diff
@@ -0,0 +1,19 @@
+import {useState} from "react";
+
+export function GenreSelect({genres, selectedGenre, onGenreSelectCb}) {
+    const [selectedGenreState, setSelectedGenreState] = useState(selectedGenre);
+
+    function onGenreSelect(e) {
+        setSelectedGenreState(e.target.value);
+        onGenreSelectCb(e);
+    }
+
+    return (
+        <div>
+            <h1>Genre Select</h1>
+            <select onChange={onGenreSelect} value={selectedGenreState}>
+                {genres.map(genre => <option key={genre} value={genre}>{genre}</option>)}
+            </select>
```

Comment:
According to the task, the list should be rendered as "genre buttons", not a select. Please rewrite the select into the list of buttons.